frontend/src/__tests__/cypress/cypress/tests/e2e/dataScienceProjects/workbenches/workbenches.cy.ts

```diff
@@ -72,7 +72,7 @@ describe('Workbench and PVSs tests', () => {
     cy.step(`Create Workbench ${projectName} using storage ${PVCDisplayName}`);
     workbenchPage.findCreateButton().click();
     createSpawnerPage.getNameInput().fill(workbenchName);
-    createSpawnerPage.findNotebookImage('jupyter-minimal-notebook').click();
+    createSpawnerPage.findNotebookImage('s2i-minimal-notebook').click();
```

Comment:
@FedeAlonso 
I ran this test against a ODH Cluster and this is failing, can you review again please? 

![image](https://github.com/user-attachments/assets/9592609c-8b78-46ef-9d79-fc773d66eb65)

Reply:
I've tested it in the PSI-6 cluster and seems to be working fine
![image](https://github.com/user-attachments/assets/0ae9a85e-8e3a-4550-a714-c7f26b6fbc60)

![image](https://github.com/user-attachments/assets/15125da2-a6a6-4f01-a754-f1aabbe2f133)

Reply:
@FedeAlonso Apologies, but can you review this again? The current checked in code looks for s2i-minimal-notebook but your screenshot shows it selected minimal python.